polkadot/roadmap/implementers-guide/src/protocol-validator-disabling.md

```diff
@@ -0,0 +1,282 @@
+
+> **Note:** \
+> There was an additional mechanism of automatically chilling the validator which removed their intent to participate in the next election, but the removed validator could simply re-register his intent to validate.
+
+There is a need to have a more immediate way to deal with malicious validators. This is where the validator disabling comes in. It is focused on dispensing **low latency** consequences for malicious actors. It is important to note that the validator disabling is not a replacement for the dispute or slashing systems. It is a complementary system that is focused on lighter but immediate consequences usually in the form of restricted validator privileges.
+
+The primary goals are:
+- Eliminate cases where attackers can get free attempts at attacking the network
+- Eliminate or minimize the risks of honest nodes being pushed out of consensus by getting unjustly slashed
+
+The above three goals are generally at odds so a careful balance has to be struck between them. We will achieve them by sacrificing some **liveness** in favor of **soundness** when the network is under stress. Maintaining  some liveness but absolute soundness ia paramount.
+
+> **Note:** \
+> Liveness = Valid candidates can go through (at a decent pace) \
+> Security = Invalid candidates cannot go through (or are statistically very improbable)
+
+Side goals are:
+- Reduce the damages to honest nodes that had a fault which might cause repeated slashes
+- Reduce liveness impact of individual malicious attackers
+
+## System Overview
+
+High level assumptions and goals of the validator disabling system that will be further discussed in the following sections:
+
+1. If validator gets slashed (even 0%) we mark them as disabled in the runtime and on the node side.
+1. We only disable up to byzantine threshold of the validators.
+1. If there are more offenders than byzantine threshold disable only the highest offenders. (Some might get re-enabled.)
+1. Disablement lasts for 1 era.
+1. Disabled validators remain in the active validator set but have some limited permissions.
+1. Disabled validators can get re-elected.
+1. Disabled validators cannot back candidates.
+1. Disabled validators can participate in approval checking.
+1. Disabled validators cannot initiate disputes, but their votes are still counted if a dispute occurs.
+1. Disabled validators making dispute statements no-show in approval checking.
+1. Disabled validators can participate in GRANDPA, but equivocations cause disablement.
+1. Disabling affects Block Authoring. (Both ways: block authoring equivocation disables and disabling stops block authoring)
```

Comment:
nit: might be better for the reader to group by what they can and can't do instead of mixing